OneWelcomeExampleApp/ViewControllers/Welcome/WelcomeViewController.swift

```diff
@@ -87,3 +89,17 @@ class WelcomeViewController: UIViewController {
         }
     }
 }
+
+private extension WelcomeViewController {
+    func setupAppIcon() {
```

Comment:
even if you find a way to change it dynamically ...

the work (this code) should be done by VC's presenter, not here